diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+1. Organizing code into modules.
+2. Setting up build tools.
+3. Client side rendering.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+The templating system in Ember is used by the router to display relevant information to the user in the view-state. Each template was written in Handlebars, and could access and manipulate properties in the View. In-built Handlebars helpers such as {{#if}} and {{#each}} were also available.
```

Comment:
Yes. In Ember 2.0, properties are generally on the Component or sometimes the Controller, and Views are no longer used.
Ember also provides add functionality to handlebars, such as the ability to link to components.
.5/1